pkg/chunked/cache_linux.go

```diff
@@ -579,50 +724,89 @@ func byteSliceAsString(b []byte) string {
 }
 
 func findTag(digest string, cacheFile *cacheFile) (string, uint64, uint64) {
```

Comment:
(AFAICS this function is now only called from tests; maybe move it to a test file.)